gp-includes/install-upgrade.php

```diff
@@ -12,6 +12,9 @@
  * @since 1.0.0
  */
 function gp_upgrade_db() {
+	require_once ABSPATH . 'wp-admin/includes/upgrade.php';
+	require_once GP_PATH . GP_INC . 'schema.php';
+
```

Comment:
Why do you need to add this, because we don't have any changes in this function?

Reply:
These requires were originally in `glotpress.php`, before the call to `gp_upgrade_db()`. In https://github.com/GlotPress/GlotPress/pull/1764/commits/8d083ea0c6c90f5f41ab03700a5ac4d0d97f13be, I moved them to here.

My reasoning for doing this was that I think the `gp_upgrade_db()` function should be callable without knowing which files need to be imported first. Because calling the function without doing these requires first would result in an error.

So it made sense to move the requires to the function, so that it's guaranteed that when the function runs, all code that the function needs is defined.

Let me know if you prefer that these requires are in `glotpress.php`.

Reply:
Something I forgot to mention:

The `gp_upgrade_db()` is called in two places now, so if the function doesn't do the requires, they need to be added twice, before each call to the function.

Alternatively, they could be added to the top level of `glotpress.php`, but that would mean that they would be imported even if there is no need to upgrade the db.